GoogleSignIn/Sources/GIDSignIn.m

```diff
@@ -882,10 +891,17 @@ - (void)addCompletionCallback:(GIDAuthFlow *)authFlow {
   [authFlow addCallback:^() {
     GIDAuthFlow *handlerAuthFlow = weakAuthFlow;
     if (self->_currentOptions.callback) {
-      GIDSignInCallback callback = self->_currentOptions.callback;
+      void (^callback)(GIDUserAuth *_Nullable userAuth, NSError *_Nullable error) = self->_currentOptions.callback;
       self->_currentOptions = nil;
       dispatch_async(dispatch_get_main_queue(), ^{
-        callback(self->_currentUser, handlerAuthFlow.error);
+        if (handlerAuthFlow.error) {
+          callback(nil, handlerAuthFlow.error);
+        } else {
+          OIDAuthState *authState = handlerAuthFlow.authState;
+          NSString * _Nullable serverAuthCode = [authState.lastTokenResponse.additionalParameters[@"server_code"] copy];
+          GIDUserAuth *  userAuth = [[GIDUserAuth alloc]initWithGoogleUser:self->_currentUser serverAuthCode:serverAuthCode];
```

Comment:
Remove spaces between `*` and `userAuth`

Reply:
Done.